web/docs/operators/chart.md

```diff
@@ -12,8 +12,11 @@ Add metadata to a schema, necessary for rendering as a chart.
 
 ```
 chart line [-x|--x-axis <fields>] [-y|--y-axis <field>]
+           [--x-axis-type=<x-axis-type>]  [--y-axis-type=<y-axis-type>]
 chart area [-x|--x-axis <fields>] [-y|--y-axis <field>]
+           [--x-axis-type=<x-axis-type>]  [--y-axis-type=<y-axis-type>]
 chart bar  [-x|--x-axis <fields>] [-y|--y-axis <field>]
+           [--x-axis-type=<x-axis-type>]  [--y-axis-type=<y-axis-type>]
 chart pie  [--name <field>] [--value <fields>]
```

Comment:
Inconsistency nitpick: we generally don't have an equal sign after long options in the docs.

This also applies to the newly added example at the bottom.

Reply:
Thanks for the 🦅 👀!